examples/functions/golang/grpc-serving/main.go

```diff
@@ -36,7 +38,7 @@ func main() {
 	})
 }
 
-func makeMatches(ctx context.Context, profile string, rosters []*pb.Roster, pools []*pb.PlayerPool) (string, []*pb.Roster, error) {
+func makeMatches(ctx context.Context, logger *log.Entry, profile string, rosters []*pb.Roster, pools []*pb.PlayerPool) (string, []*pb.Roster, error) {
```

Comment:
I wanted to do that but its a little more restructuring - since the apisrv and the main need it - harness is a pass through - so it needs to be in a separate module then. (Cant be in harness as that will have a cyclic dependency with apisrv then).

I propose having this go through and then separate it out as a part of refactoring the harness (proposal is to have a shared harness in serving package - and then packages for required structures for evaluator / match functions)